string path_name
float64[] velocity  #optional
float64[] acceleration #optional
---
string message
